modules/network-observability-total-resource-usage.adoc

```diff
@@ -0,0 +1,44 @@
+// Module included in the following assemblies:
+// * network_observability/configuring_operator.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="network-observability-total-resource-usage-table_{context}"]
+= Total average memory and CPU usage
+The following table outlines averages of total resource usage for clusters with a sampling value of 1, 50, and 400.
+
+[IMPORTANT]
+====
+The examples outlined in the table demonstrate scenarios that are tailored to specific workloads. Consider each example only as a baseline from which adjustments can be made to accommodate your workload needs.
+====
+
+.Total average resource usage
+[%autowidth, options="header"]
+|===
+| Sampling value | Parameters | Test 1 (25 nodes) ^[1]^ | Test 2 (65 nodes) ^[2]^ | Test 3 (120 nodes) ^[3]^
+.6+| *Sampling = 1* | *With Loki* 3+|
+| Total NetObserv CPU Usage | 3.24 | 3.42 | 7.30
+| Total NetObserv RSS (Memory) Usage | 14.09 GB | 11.81 GB | 36.46 GB
+| *Without Loki* 3+|
+| Total NetObserv CPU Usage | 2.40 | 2.43 | 5.59
+| Total NetObserv RSS (Memory) Usage | 6.85 GB | 10.39 GB | 13.92 GB
+.6+| *Sampling = 50* | *With Loki* 3+|
+| Total NetObserv CPU Usage | 2.04 | 2.36 | 3.31
+| Total NetObserv RSS (Memory) Usage | 8.79 GB | 19.14 GB | 21.07 GB
+| *Without Loki* 3+|
+| Total NetObserv CPU Usage | 1.55 | 1.64 | 2.70
+| Total NetObserv RSS (Memory) Usage | 6.71 GB | 10.15 GB | 14.82 GB
+.6+| *Sampling = 400* | *With Loki* 3+|
+| Total NetObserv CPU Usage | 1.71 | 1.44 | 2.36
+| Total NetObserv RSS (Memory) Usage | 8.21 GB | 16.02 GB | 17.44 GB
+| *Without Loki* 3+|
+| Total NetObserv CPU Usage | 1.31 | 1.06 | 1.83
+| Total NetObserv RSS (Memory) Usage | 7.01 GB | 10.70 GB | 13.26 GB
+|===
+
+--
+1. Test 1 focuses on pods density per node, each pod generating low amounts of traffic on a small size cluster.
+2. Test 2 focuses on ingress traffic throughput on a medium size cluster.
+3. Test 3 focuses on pods density, generating heavy traffic on a large size cluster.
```

Comment:
in fact we shouldn't say "heavy traffic" for Test 3, it's not heavy. I think at first my suggestion was to just say "more traffic than in Test 1"